diff --git a/wsd/FileServer.cpp b/wsd/FileServer.cpp
index f524cdb9b0..e25eed6325 100644
--- a/wsd/FileServer.cpp
+++ b/wsd/FileServer.cpp
@@ -1376,7 +1376,7 @@ std::string FileServerRequestHandler::getRequestPathname(const HTTPRequest& requ
 
     std::string path(requestUri.getPath());
 
-    Poco::RegularExpression gitHashRe("/([0-9a-f]+)/");
+    Poco::RegularExpression gitHashRe("/([0-9a-f.]+)/");
     std::string gitHash;
     if (gitHashRe.extract(path, gitHash))
     {
